src/libraries/System.Private.Xml/tests/XmlWriter/System.Xml.RW.XmlWriter.Tests.csproj

```diff
@@ -7,5 +7,6 @@
     <Compile Include="WriteWithEncoding.cs" />
     <Compile Include="WriteWithEncodingWithFallback.cs" />
     <Compile Include="WriteWithInvalidSurrogate.cs" />
+    <Compile Include="XmlTextWriterTests.cs" />
```

Comment:
this caught my eye :/ .. as far as I can see our test coverage of XmlTextWriter is minimal. Our tests test various other `*Xml*Writer`s. XmlWriter.Create() doesn't seem to create one either. Not an urgent problem, but I suppose it led to this bug.